docs/reference/buildx_bake.md

```diff
@@ -339,7 +339,7 @@ target "db" {
 
 Complete list of valid target fields:
 
-`args`, `cache-from`, `cache-to`, `context`, `dockerfile`, `inherits`, `labels`,
```

Comment:
Could we have an example with named `contexts` on this page? Also clarify difference with `context` would be sweet.

Reply:
I need examples for the follow-up target linking anyway. Maybe better to do them together. WIP in https://github.com/docker/buildx/compare/master...tonistiigi:bake-named-contexts-2?expand=1